rasa/core/policies/embedding_policy.py

```diff
@@ -1,694 +1,139 @@
-        "pos_encoding": "timing",  # string 'timing' or 'emb'
+        NUM_TRANSFORMER_LAYERS: 1,
         # max sequence length if pos_encoding='emb'
-        "max_seq_length": 256,
+        MAX_SEQ_LENGTH: 256,
         # number of attention heads in transformer
-        "num_heads": 4,
+        NUM_HEADS: 4,
         # training parameters
         # initial and final batch sizes:
         # batch size will be linearly increased for each epoch
-        "batch_size": [8, 32],
+        BATCH_SIZES: [8, 32],
         # how to create batches
-        "batch_strategy": "balanced",  # string 'sequence' or 'balanced'
+        BATCH_STRATEGY: "balanced",  # string 'sequence' or 'balanced'
         # number of epochs
-        "epochs": 1,
+        EPOCHS: 1,
         # set random seed to any int to get reproducible results
-        "random_seed": None,
+        RANDOM_SEED: None,
         # embedding parameters
         # dimension size of embedding vectors
-        "embed_dim": 20,
+        EMBED_DIM: 20,
         # the type of the similarity
-        "num_neg": 20,
+        NUM_NEG: 20,
         # flag if minimize only maximum similarity over incorrect labels
-        "similarity_type": "auto",  # string 'auto' or 'cosine' or 'inner'
+        SIMILARITY_TYPE: "auto",  # string 'auto' or 'cosine' or 'inner'
         # the type of the loss function
-        "loss_type": "softmax",  # string 'softmax' or 'margin'
+        LOSS_TYPE: "softmax",  # string 'softmax' or 'margin'
         # number of top actions to normalize scores for softmax loss_type
         # set to 0 to turn off normalization
-        "ranking_length": 10,
+        RANKING_LENGTH: 10,
         # how similar the algorithm should try
         # to make embedding vectors for correct labels
-        "mu_pos": 0.8,  # should be 0.0 < ... < 1.0 for 'cosine'
+        MAX_POS_SIM: 0.8,  # should be 0.0 < ... < 1.0 for 'cosine'
         # maximum negative similarity for incorrect labels
-        "mu_neg": -0.2,  # should be -1.0 < ... < 1.0 for 'cosine'
+        MAX_NEG_SIM: -0.2,  # should be -1.0 < ... < 1.0 for 'cosine'
         # the number of incorrect labels, the algorithm will minimize
         # their similarity to the user input during training
-        "use_max_sim_neg": True,  # flag which loss function to use
+        USE_MAX_NEG_SIM: True,  # flag which loss function to use
         # scale loss inverse proportionally to confidence of correct prediction
-        "scale_loss": True,
+        SCALE_LOSS: True,
         # regularization
-        # the scale of L2 regularization
-        "C2": 0.001,
+        # the scale of regularization
+        REGULARIZATION_CONSTANT: 0.001,
         # the scale of how important is to minimize the maximum similarity
         # between embeddings of different labels
-        "C_emb": 0.8,
+        NEG_MARGIN_SCALE: 0.8,
         # dropout rate for dial nn
-        "droprate_a": 0.1,
+        DROPRATE_DIALOGUE: 0.1,
```

Comment:
there is a space in `drop rate` / `drop out rate`, isn't it?
```suggestion
        DROP_RATE_DIALOGUE: 0.1,
```

Reply:
dropout without a space